src/components/views/rooms/RoomListPanel/RoomListHeaderView.tsx

```diff
@@ -69,12 +69,14 @@ export function RoomListHeaderView(): JSX.Element {
                         {vm.displayComposeMenu ? (
                             <ComposeMenu vm={vm} />
                         ) : (
-                            <IconButton
-                                aria-label={_t("action|start_chat")}
-                                onClick={(e) => vm.createChatRoom(e.nativeEvent)}
-                            >
-                                <ComposeIcon color="var(--cpd-color-icon-secondary)" />
-                            </IconButton>
+                            <Tooltip label={_t("action|start_chat")}>
+                                <IconButton
```

Comment:
It looks like IconButton has a `tooltip` prop you can use, which would also cover the aria-label I believe